test/mitmproxy/utils/test_data.py

```diff
@@ -1,7 +1,8 @@
+import pytest
 from mitmproxy.utils import data
-from . import tutils
 
 
 def test_pkg_data():
     assert data.pkg_data.path("tools/console")
-    tutils.raises("does not exist", data.pkg_data.path, "nonexistent")
```

Comment:
Do we still have a use for our custom `tutils.raises` handler?
Maybe we should ditch it everywhere and move to `pytest.raises`?